### PR TITLE
CNV-12361: Adding Quick Start section to New Features in RN

### PR DESCRIPTION
**Pull request by bgaydosrh**
for 4.8 only.

Jira: https://issues.redhat.com/browse/CNV-12361

Tagging @matthewcarleton for code review.
Tagging @duyanyan for QE review.

Direct doc preview link: https://deploy-preview-34299--osdocs.netlify.app/openshift-enterprise/latest/virt/virt-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#virt-4-8-quick-starts

### CODE REVIEW

**Comment by gouyang** [comment]
LGTM.

File: virt/virt-4-8-release-notes.adoc [hunk 1]
```diff
@@ -53,6 +53,11 @@ The SVVP Certification applies to:
 ** Storage
 ** Guest memory swapping
 
+[id="virt-4-8-quick-starts"]
+=== Quick starts
+
+Quick start tours are available for several {VirtProductName} features. To view the tours, click the *?* icon in the masthead of the {VirtProductName} console and select *Quick Starts*.
```
Comment (matthewcarleton) [inline virt/virt-4-8-release-notes.adoc:59]: Can we add further instructions on how to filter the page to show just the virt quick starts?

Comment (matthewcarleton) [reply virt/virt-4-8-release-notes.adoc:59]:
```suggestion
Quick start tours are available for several {VirtProductName} features. To view the tours, click the *?* icon in the masthead of the {VirtProductName} console and select *Quick Starts*.
You can filter the available quickstarts with the `virtualization` keyword in the filter field.
```

**Comment by bgaydosrh** [comment]
Closing pull request to get a fresh copy of the notes.

File: virt/virt-4-8-release-notes.adoc (hunk 1 above)
Comment (bgaydosrh) [reply virt/virt-4-8-release-notes.adoc:59]: Added in new PR: https://github.com/openshift/openshift-docs/pull/34611

**Comment by bgaydosrh** [comment]
This PR replaced by https://github.com/openshift/openshift-docs/pull/34611

**Pull request closed without merging**